src/components/tile.jsx

```diff
@@ -0,0 +1,40 @@
+import brand from "../assets/brand.png"
+import story from "../assets/our_story.png"
+
+export default function Tile(props) {
+    let image;
+    if (props.img === "brand") {
+        image = brand
+    } else {
+        image = story
+    }
+
+    if (props.img !== undefined) {
```

Comment:
In plaat van het if-else statement hier kun je gebruik maken van de children-property. Zie edhub react fundamentals 4.6. 

In je Tile-component krijg je dan:
```suggestion
    function Tile(props) {
    
    return (
        <>
            <section>
                <img src={props.image} alt={props.img} />
                <h2>{props.title}</h2>
                {props.children}
            </section>
        </>    
    );
}

export default Tile;
```

en in App.jsx krijg je dan:
```suggestion
          <footer>
                <Tile title="THE BRAND">
                     <p> {lorem.brandp1} </p>
                      <p> {lorem.brandp2} </p>
                </Tile>      
                <Tile image={brand} img="brand" > <Tile/>
                <Tile image={story} img="story"> <Tile/>
                <Tile title={"OUR STORY"}>
                      <p>{lorem.story} </p>
                </Tile>      
      </footer>
```